edward/inferences.py

```diff
@@ -238,14 +246,14 @@ def print_progress(self, t, loss):
                 print(self.variational)
 
     def finalize(self):
```

Comment:
If the user is in the setting of preloaded data or reading from files, it could still be useful to continue using the same threads after interrupting computation.

I guess if there are _no_ queues we could have it not use a coordinator and queue runner. Tracking this would require storing an additional class attribute that tracks whether any of the data values are not placeholders. I think this could add a complication that may lead to harder debugging as one must keep this in mind. It's easier to think of the default of _always_ using a queue runner.